api/app/tests/wildfire_one/test_validation.py

```diff
@@ -0,0 +1,192 @@
+    _, high_rh_valid, _, _, _ = get_valid_flags(high_valid)
+    assert high_rh_valid is True
+
+
+def test_rh_invalid():
+    """ Below 0 and above 100 is invalid for rh"""
+    low_valid = WeatherReading(temperature=None,
+                               relative_humidity=-1,
+                               wind_speed=None,
+                               wind_direction=None,
+                               precipitation=None)
+    _, low_rh_invalid, _, _, _ = get_valid_flags(low_valid)
+    assert low_rh_invalid is False
+
+    high_valid = WeatherReading(temperature=None,
+                                relative_humidity=101,
+                                wind_speed=None,
+                                wind_direction=None,
+                                precipitation=None)
+    _, high_rh_invalid, _, _, _ = get_valid_flags(high_valid)
+    assert high_rh_invalid is False
+
+
+def test_wind_speed_valid():
+    """ 0 to inf is valid for wind_speed"""
+    low_valid = WeatherReading(temperature=None,
+                               relative_humidity=None,
+                               wind_speed=0,
+                               wind_direction=None,
+                               precipitation=None)
+    _, _, low_wind_speed_valid, _, _ = get_valid_flags(low_valid)
+    assert low_wind_speed_valid is True
+
+    high_valid = WeatherReading(temperature=None,
+                                relative_humidity=None,
+                                wind_speed=math.inf,
+                                wind_direction=None,
+                                precipitation=None)
+    _, _, high_wind_speed_valid, _, _ = get_valid_flags(high_valid)
+    assert high_wind_speed_valid is True
+
+
+def test_wind_speed_invalid():
+    """ Below 0 is invalid for wind_speed"""
+    low_valid = WeatherReading(temperature=None,
+                               relative_humidity=None,
+                               wind_speed=-1,
+                               wind_direction=None,
+                               precipitation=None)
+    _, _, low_wind_speed_invalid, _, _ = get_valid_flags(low_valid)
+    assert low_wind_speed_invalid is False
+
+
+def test_wdir_valid():
+    """ 0 to 360 is valid for wdir"""
+    low_valid = WeatherReading(temperature=None,
+                               relative_humidity=None,
+                               wind_speed=None,
+                               wind_direction=0,
+                               precipitation=None)
+    _, _, _, low_wdir_valid, _ = get_valid_flags(low_valid)
+    assert low_wdir_valid is True
+
+    high_valid = WeatherReading(temperature=None,
+                                relative_humidity=None,
+                                wind_speed=None,
+                                wind_direction=360,
+                                precipitation=None)
+    _, _, _, high_wdir_valid, _ = get_valid_flags(high_valid)
+    assert high_wdir_valid is True
```

Comment:
I would've expected that only one of 0 or 360 would be valid, since 0 degrees = 360 degrees. Might not actually matter though (unless we implement wind roses at some point, maybe?)

Reply:
I didn't change the logic of validity, just added tests.